arlas-server/src/main/java/io/arlas/server/app/ArlasServer.java

```diff
@@ -171,6 +172,11 @@ public void run(ArlasServerConfiguration configuration, Environment environment)
             LOGGER.info("Explore API disabled");
         }
 
+        // Auth
+        if (configuration.arlasAuthConfiguration.enabled) {
```

Comment:
what if `configuration.arlasAuthConfiguration` is null?

Reply:
Good point. I'll initialize it with enabled=false if null.